### DISCUSSION TITLE
Error to install securityonion Latest version: 2.4.110

### DISCUSSION THREAD

**Discussion by lenovomy** [Security-Onion-Solutions/securityonion#13939] [2.4]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

30

### Storage for /

400GB

### Storage for /nsm

400GB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

hi iHello, I installed version 2.4.110 in Standalone and Distributed modes, but unfortunately I encountered errors in two cases.
![Error](https://github.com/user-attachments/assets/022eb9cd-3ff6-465d-a34e-9b2b79af0d1a)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by rh-ops** [comment]
Good morning. Can you relay what the log in the screenshot says?

`cat /root/sosetup.log`

**Comment by lenovomy** [comment]
hi thanks
Which part of the sosetup file should I send you a screenshot of?
plz lookat atachment file

**Reply by rh-ops** [reply]
Correction, lets look at a different log. this one is much shorter so everything in it should fit in 1 screenshot. Cat /root/errors.log.
